database/xc7z/settings.sh

```diff
@@ -0,0 +1,29 @@
+# Unsetting all previous XRAY env variables
+unset ${!XRAY_@}
+
+export XRAY_DATABASE="xc7z"
+export XRAY_PART="xc7z010clg400-1"
+export XRAY_ROI_FRAMES="0x00000000:0xffffffff"
+export XRAY_XC7Z="1"
```

Comment:
Earlier comment on this still stands. This is redundant with XRAY_DATABASE

Reply:
Actually, if we're about to drop DEBUG bitstream and use PERFRAMECRC for all the families this is not necessary.

Reply:
The name has been changed to zynq. XRAY_XC7Z is removed.